MAUI/RadialGauge/shape-pointer.md

```diff
@@ -0,0 +1,244 @@
+---
+layout: post
+title: Shape Pointer in .NET MAUI Radial Gauge control | Syncfusion
+description: Learn here all about Shape Pointer feature of Syncfusion .NET MAUI Radial Gauge control with in-build shapes.
+platform: maui
+control: SfRadialGauge
+documentation: ug
+---
+
+# Shape Pointer in .NET MAUI Radial Gauge
+
+Indicate current values by using different types of shape pointers. You can change the shape type using the [`ShapeType`]() property.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<gauge:SfRadialGauge>
+    <gauge:SfRadialGauge.Axes>
+        <gauge:RadialAxis>
+            <gauge:RadialAxis.Pointers>
+                <gauge:ShapePointer Value="60" />
+            </gauge:RadialAxis.Pointers>
+        </gauge:RadialAxis>
+    </gauge:SfRadialGauge.Axes>
+</gauge:SfRadialGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfRadialGauge sfRadialGauge = new SfRadialGauge();
+
+RadialAxis radialAxis = new RadialAxis();
+sfRadialGauge.Axes.Add(radialAxis);
+
+ShapePointer pointer = new ShapePointer();
+pointer.Value = 60;
+radialAxis.Pointers.Add(pointer);
+
+this.Content = sfRadialGauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Radial Gauge Default Shape Pointer](images/marker-pointers/maui-radial-gauge-default-marker-pointer.PNG)
+
+Gauge supports the following types of shapes:
+
+* Circle
+* Diamond
+* Inverted triangle
+* Rectangle
+* Triangle
+
+![.NET MAUI Radial Gauge Markers](images/marker-pointers/maui-radial-gauge-markers.PNG)
+
+## Shape customization
+
+The shape pointer can be customized using the following properties:
+
+* [`Fill`]() – Allows you to customize the shape color.
+* [`ShapeHeight`]() – Allows you to specify the shape height.
+* [`ShapeWidth`]() – Allows you to specify the shape width.
+* [`Stroke`]() – Allows you to specify the border color for the shape.
+* [`BorderWidth`]() –  Allows you to specify the border width of the shape.
+* [`HasShadow`]() –  Allows you to specify the shadow of the shape.
+* [`OverlayFill`]() –  Allows you to specify the color for dragging overlay for the shape.
+* [`OverlayRadius`]() –  Allows you to specify the radius for dragging overlay for the shape.
```

Comment:
Extra spacing

Allows you to specify the radius for the shapes' dragging overlay.